### DISCUSSION TITLE
LightningCSS with NextJS 14.04

### DISCUSSION THREAD

**Discussion by paulm17** [parcel-bundler/lightningcss#645] [Ideas]
Now that 14.04 has been released, I saw this: https://github.com/vercel/next.js/pull/58471

Does this mean that lightningcss can now be used in lieu of postcss?

Would be nice if there is some documentation on how to get this setup.  Or perhaps to wait until Turbopack is finalised?  I don't know.

Anyway, just wanted to bring it up.  Thoughts?  🤔 

Thanks!

**Comment by devongovett** [comment]
This seems like a question for the Next.js repo. I'm not really sure about their plans either.
